TLC5955.h

```diff
@@ -103,26 +107,22 @@ uint32_t getGsclkFreq();
 /* Diagnostic Methods */
 void printByte(uint8_t myByte);
 
-// uint8_t _leds_per_chip = LEDS_PER_CHIP;
-// uint8_t _color_channel_count = COLOR_CHANNEL_COUNT;
-// uint8_t _tlc_count = TLC_COUNT;
-//
-// uint8_t _dc_data[TLC_COUNT][LEDS_PER_CHIP][COLOR_CHANNEL_COUNT];
-// uint8_t _rgb_order[TLC_COUNT][LEDS_PER_CHIP][COLOR_CHANNEL_COUNT];
-// uint16_t _grayscale_data[TLC_COUNT][LEDS_PER_CHIP][COLOR_CHANNEL_COUNT];
-
 static const uint8_t _tlc_count; // This
 static const uint8_t COLOR_CHANNEL_COUNT = 3;
 static const uint8_t LEDS_PER_CHIP = 16;
 static bool enforce_max_current;
 static float max_current_amps;
 
-static uint8_t _dc_data[][LEDS_PER_CHIP][COLOR_CHANNEL_COUNT];
 static uint8_t _rgb_order[][LEDS_PER_CHIP][COLOR_CHANNEL_COUNT];
 static uint16_t _grayscale_data[][LEDS_PER_CHIP][COLOR_CHANNEL_COUNT];
 
 uint8_t rgb_order_default[3] = {0, 1, 2};
 
+  // Analog Control Values
+  // https://www.ti.com/lit/ds/symlink/tlc5955.pdf?ts=1636036806528&ref_url=https%253A%252F%252Fwww.google.com%252F
```

Comment:
```suggestion
  // https://www.ti.com/lit/ds/symlink/tlc5955.pdf
```

We definitely don't need to be tracked by people when we share links :D